src/index.html

```diff
@@ -1,13 +1,426 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
You forgot to add favicon to your site and a title for the whole page

Reply:
done